linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/AdminHandler.scala

```diff
@@ -74,6 +74,7 @@ object AdminHandler extends HtmlView {
           <title>linkerd admin</title>
```

Comment:
in this file, `<li><a href="admin/logging">logging</a></li>` on line 34 needs to be updated